README.rst

```diff
@@ -0,0 +1,63 @@
+# reana-auth-krb5
+
+Required information and inputs:
+
+
+
+## Building the image
+
+No build arguments needed.
+
+
+===========================
+REANA Environment Kerberos5
+===========================
+
+About
+=====
+
+``reana-auth-krb5`` provides a container image with encapsulated runtime
+execution environment for creating an renewing Kerberos tokens. The container image
+includes no additional logic or libraries, just the bare minimum to support the Kerberos operations.
+
+``reana-auth-krb5`` was developed for use in the `REANA <http://reana.readthedocs.io/>`_ reusable research data analysis platform.
+
+Usage
+=====
+
+You can use ``reana-auth-krb5`` as a base image, however it was built
+to be used as a sidecar container with the single purpose of refreshing the Kerberos token for long running jobs.
+Once obtained, the token is shared with the main container through the token cache directory (``KRB5CCNAME``).
+
+Some concrete usage examples will be provided later on.
```

Comment:
Ah, it is here :) Perhaps rephrase so that REANA users won't worry that they have to study `reana-auth-krb5`.  Something like "The `reana-auth-krb5` image is used internally in the REANA platform to provide ..."

Also for the usage examples, you can put "The end users can ask for Kubernetes authentication by means of declaring `kerberos: true`..." and point to the docs.reana.io section that explains how to do this.

Reply:
Done. I've also created a section for config, since the part that was talking about configuration seemed a bit misplaced next to the text for the user. Let me know what you think :)